### PR TITLE
added og tags - Issue: #103

### PR DESCRIPTION
**Pull request by tolgaouz**

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -11,6 +11,14 @@
     <meta name="msapplication-TileColor" content="#2b5797">
     <meta name="theme-color" content="#ffffff">
 
+    <meta property="og:image" content="{{ page.meta_img }}">
+    <meta property="og:image:width" content="1103">
+    <meta property="og:image:height" content="577">
+    <meta property="og:url" content="https://0x4447.com/home">
```
Comment (davidgatti) [inline _includes/head.html:17]: Eh eh eh, If I don't check this small things ;) hard coded URL? I don't think this is good.

Comment (davidgatti) [reply _includes/head.html:17]: @tolgaouz 🖐️

**Comment by tolgaouz** [comment]
Sorry for the stupid mistake :/, fixed

**Comment by davidgatti** [comment]
Not sure this makes sense. can't you construct the URL dynamically, I don't want to write one each time.

**Comment by tolgaouz** [comment]
sure thing, I thought you wanted it to be a variable

**Pull request merged**